assets/gql/contact_wa_groups/update_wa_group.gql

```diff
@@ -0,0 +1,11 @@
+mutation UpdateWaGroupContacts($input: WaGroupContactsInput!) {
+  updateWaGroupContacts(input: $input) {
```

Comment:
is it updateWaGroupContacts or updateContactWaGroups, maintain consistency in naming